Service/EventLogCleanup.php

```diff
@@ -92,10 +103,16 @@ public function deleteEventLogEntries(int $daysOld, ?int $campaignId, bool $dryR
             $operations[self::EMAIL_STATS]         = true;
         }
 
+        if (array_key_exists(self::EMAIL_STATS_TOKENS, $operations) && true === $operations[self::EMAIL_STATS_TOKENS]) {
+            unset($operations[self::EMAIL_STATS_TOKENS]);
+            $operations[self::EMAIL_STATS_TOKENS]  = true;
+        }
+
```

Comment:
Not sure about what's happening here, please explain sense:

```
if:
  EMAIL_STATS_TOKENS key exists in $operations array AND array value of EMAIL_STATS_TOKENS key is TRUE
then:
  unset value of array key EMAIL_STATS_TOKENS;
  set value of array key EMAIL_STATS_TOKENS to TRUE again;
```

Reply:
Talked with @Moongazer about this.

Result: Thanks @Moongazer for the recommendation, that this part is not necessarrely needed (didn´t realized it in the first step) - so I removed this not harmful but not necessary part with my current commit.